README.md

```diff
@@ -5,4 +5,36 @@
   - `docsRoot/data/File.yaml` becomes `.Site.Data.File.DataField.SubField[index].Etc`
 - Specifically, this theme uses the following files/parameters
 - `data/Solo.yaml` (generated by docs build script in docs dir)
+  - DocsVersion: either has the form of "/product/version" or "" empty string
```

Comment:
can we do another bullet point, be more explicit about what `version` and `product` might look like?

e.g. `/gloo/0.21.0` vs `/gloo/hash` vs `/gloo/0.21.0-hash`?